Engine/Component/ComponentParticleSystem.cpp

```diff
@@ -318,12 +341,14 @@ void ComponentParticleSystem::SpecializedLoad(const Config& config)
 	color_particle[3] = config.GetFloat("Color Particle A", 1.0F);
 	fade = config.GetBool("Fade", false);
 	fade_time = config.GetFloat("Fade Time", 1.0F);
+	size_change_speed = config.GetFloat("Fade Time", 1.0F);
 	color_fade_time = config.GetFloat("Color Fade Time", 1.0F);
 	fade_between_colors = config.GetBool("Fade between Colors", false);
 	color_to_fade[0] = config.GetFloat("Color to fade R", 1.0F);
 	color_to_fade[1] = config.GetFloat("Color to fade G", 1.0F);
 	color_to_fade[2] = config.GetFloat("Color to fade B", 1.0F);
 	color_to_fade[3] = config.GetFloat("Color to fade A", 1.0F);
+	billboard->SpecializedLoad(config);
```

Comment:
maybe the specializedload and save can go at the end of the file since they are a super long methods and to scroll down to the other functions takes a little bit long, but just personal taste